app.js

```diff
@@ -1,17 +1,57 @@
 const path = require('path');
-
 const express = require('express');
 const bodyParser = require('body-parser');
 
+const multer = require('multer');
+const port = process.env.PORT || 3000;
+
 const errorController = require('./controllers/error');
 
 const app = express();
+app.use(express.static(__dirname + '/public'));
+const storage = multer.diskStorage({
+    destination: function(req, file, cb) {
+        cb(null, 'uploads/');
+    },
+
+    // By default, multer removes file extensions so let's add them back
+    filename: function(req, file, cb) {
+        cb(null, file.fieldname + '-' + Date.now() + path.extname(file.originalname));
+    }
+});
+app.listen(port, () => console.log(`Listening on port ${port}...`));
 
 app.set('view engine', 'ejs');
 app.set('views', 'views');
 
 const adminRoutes = require('./routes/admin');
 const blogRoutes = require('./routes/blog');
+const helpers = require('./helpers');
+app.post('/upload-profile-pic', (req, res) => {
```

Comment:
toata functia asta cu app.post('/upload-...) ar fi bine pe viitor sa o muti si pe asta in routes/blog.js sau cum ii zice, unde se fac toate verificarile de rute.